.github/workflows/test.yml

```
@@ -13,6 +13,7 @@ jobs:
         ocaml-compiler:
           - 5.1.x
           - 5.2.x
+          - 5.3.0~beta2
```

i think your version of setup-ocaml is a bit out-of-date so this may or may not work
```suggestion
          - ocaml-base-compiler.5.3.0~beta2
```

---

rebased